src/screens/rewards/components/RewardsTotalEarnings.tsx

```diff
@@ -42,11 +42,7 @@ export const RewardsTotalEarnings: React.FC<Props> = ({
               <Text color="labelTertiary" size="15pt" weight="semibold">
                 {i18n.t(i18n.l.rewards.multiplier)}
               </Text>
-              <Text
-                size="26pt"
-                color={{ custom: colors.networkColors.optimism }}
-                weight="bold"
-              >
+              <Text size="26pt" color={{ custom: color }} weight="bold">
```

Comment:
just for convenience when you have a lot of RDS `color={{ custom: color }}` props you could just wrap everything in `<AccentColorProvider color={color} />` and use color="accent" instead